calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx

```diff
@@ -0,0 +1,58 @@
+---
+description: Create and assign custom roles to give specific, cluster-specific permission to your users.
+---
+
+import IconUser from '/img/icons/user-icon.svg';
+
+# Creating and assigning custom roles
+
+As an administrator, you can create custom, cluster-specific roles that restrict users to particular functions on a cluster.
+
+## Overview
+
+Calico Cloud comes with a set of predefinied global roles that let you give permissions to users based on what they need to do.
+For example, a user with the Security global role has a broader set of permissions than a user with the Viewer global role.
+These permissions apply to all clusters.
+
+But in some cases these global roles can be too broad.
+
+By creating and assigning custom roles, administrators can be much more discriminating about what permissions they give users.
```

Comment:
Doc point of view is uneven. Doc goes back and forth between "you" and "administrators" and "to your users". Pick one view and apply.

Reply:
done